examples/campaign_management/create_campaign_experiment.py

```diff
@@ -0,0 +1,110 @@
+#!/usr/bin/env python
+# Copyright 2019 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Demonstrates how to begin creation of a campaign experiment from a draft
+and wait for it to complete."""
+
+from __future__ import absolute_import
+
+import argparse
+import six
+import sys
+import uuid
+
+import google.ads.google_ads.client
+
+
+def main(client, customer_id, campaign_draft_resource_name):
+    # Create the campaign experiment.
+    campaign_experiment = client.get_type('CampaignExperiment', version='v1')
+    campaign_experiment.campaign_draft.value = campaign_draft_resource_name
+    campaign_experiment.name.value = 'Campaign Experiment #{}'.format(
+        uuid.uuid4())
+    campaign_experiment.traffic_split_percent.value = 50
+    campaign_experiment.traffic_split_type = client.get_type(
+        'CampaignExperimentTrafficSplitTypeEnum',version='v1').RANDOM_QUERY
+
+    try:
+        campaign_experiment_service = client.get_service(
+            'CampaignExperimentService',
+            version='v1'
+        )
+
+        # A Long Running Operation (LRO) is returned from this
+        # asynchronous request by the API.
+        campaign_experiment_lro = (
+            campaign_experiment_service.create_campaign_experiment(
+                customer_id,
+                campaign_experiment
+            )
+        )
+    except google.ads.google_ads.errors.GoogleAdsException as ex:
+        print('Request with ID "%s" failed with status "%s" and includes the '
```

Comment:
Throughout the example can you change instances of string interpolation that use `'%s' % s` to instead use `'{}'.format(s)`?

This print statement would then look like:

```python
print('Request with ID "{}" failed with status "{}" and includes the '
      'following errors:'.format(ex.request_id, ex.error.code().name))
```

Reply:
Done